### PR TITLE
Update rubygem-foreman_ansible to 6.3.0

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 1]
```diff
@@ -42,8 +46,8 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 9.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 6.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 7.0.0
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec:50]: this won't work ;)

File: packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 2]
```diff
@@ -21,14 +21,18 @@ Requires: foreman >= %{foreman_min_version}
...
+Requires: %{?scl_prefix}rubygem(acts_as_list) < 1.1
 Requires: %{?scl_prefix}rubygem(deface) < 2.0
-Requires: %{?scl_prefix}rubygem(foreman_remote_execution) >= 4.2.0
+Requires: %{?scl_prefix}rubygem(foreman_remote_execution) >= 4.4.0
 Requires: %{?scl_prefix}rubygem(ipaddress) >= 0.8.0
 Requires: %{?scl_prefix}rubygem(ipaddress) < 1.0
 BuildRequires: foreman-assets >= %{foreman_min_version}
 BuildRequires: foreman-plugin >= %{foreman_min_version}
+BuildRequires: %{?scl_prefix}rubygem(acts_as_list) >= 1.0.3
+BuildRequires: %{?scl_prefix}rubygem(acts_as_list) < 1.1
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec:33]: needs packaging?

File: packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 3]
```diff
@@ -21,14 +21,18 @@ Requires: foreman >= %{foreman_min_version}
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(acts_as_list) >= 1.0.3
+Requires: %{?scl_prefix}rubygem(acts_as_list) < 1.1
 Requires: %{?scl_prefix}rubygem(deface) < 2.0
-Requires: %{?scl_prefix}rubygem(foreman_remote_execution) >= 4.2.0
+Requires: %{?scl_prefix}rubygem(foreman_remote_execution) >= 4.4.0
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec:27]: needs packaging

Comment (xprazak2) [reply packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec:27]: https://github.com/theforeman/foreman-packaging/pull/6623

File: packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec (hunk 2 above)
Comment (xprazak2) [reply packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec:33]: Already done: https://github.com/theforeman/foreman-packaging/tree/rpm/2.5/packages/plugins/rubygem-acts_as_list

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
closure expected

**Pull request merged**